src/calculateRentalCost.js

```diff
@@ -4,7 +4,7 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  console.log('No solution at all');
 }
```

Comment:
The function `calculateRentalCost` currently does not implement the required logic as described in the task. You need to calculate the total cost based on the number of days the car is rented, applying discounts for rentals of 3 or more days, and 7 or more days. Please implement the function using constants for the daily cost and the discounts, as per the checklist items 2 and 4.